go/libraries/doltcore/sqle/dfunctions/dolt_merge_base.go

```diff
@@ -27,11 +27,13 @@ import (
 
 const DoltMergeBaseFuncName = "dolt_merge_base"
 
+// Deprecated: please use the version in the dprocedures package
 type MergeBase struct {
 	expression.BinaryExpression
 }
 
 // NewMergeBase returns a MergeBase sql function.
+// Deprecated: please use the version in the dprocedures package
 func NewMergeBase(left, right sql.Expression) sql.Expression {
```

Comment:
This one should not be deprecated: it's read-only (an actual function, not a procedure)